axelrod/strategies/cycler.py

```diff
@@ -72,9 +72,11 @@ def __init__(self, cycle="CCD") -> None:
         self.classifier['memory_depth'] = len(cycle) - 1
 
     def strategy(self, opponent: Player) -> Action:
+        actions = {'C': Actions.C, 'D': Actions.D}
         curent_round = len(self.history)
         index = curent_round % len(self.cycle)
-        return self.cycle[index]
+        action_str = self.cycle[index]
+        return actions[action_str]
```

Comment:
This could all be refactored used `itertools.cycle` (as in the `axelrod.MockPlayer`). Since you've refactored the strategy I suggest doing it that way (removes the need for index etc).